index.bs

```diff
@@ -44,13 +44,17 @@ urlPrefix: https://tools.ietf.org/html/draft-ietf-httpbis-header-structure; spec
             text: boolean; url: #section-3.9
     type: abstract-op
         text: serialize Structured Header; url: #section-4.1
+urlPrefix: https://html.spec.whatwg.org/multipage/browsing-the-web.html;
+    type: dfn
+        text: create navigation params by fetching; url: #create-navigation-params-by-fetching
```

Comment:
Seems like this is not exported on purpose: https://github.com/whatwg/html/issues/9159#issuecomment-1506749042